kfs-cam/src/main/java/org/kuali/kfs/module/cam/businessobject/AssetBase.java

```diff
@@ -2170,4 +2172,19 @@ public String getCamsComplexMaintenanceDocumentLookup() {
         return managedLists;
     }
 
+    public List<Note> getBoNotes() {
+        if (!StringUtils.isEmpty(getObjectId())) {
+            boNotes = SpringContext.getBean(NoteService.class).getByRemoteObjectId(getObjectId());

```

Comment:
The coding standards only utilize the SpringContext when services cannot be injected. In this way, then SpringContext.getBean() should only be utilized when a service cannot be injected which is not the case here since the NoteService class can be generated by Spring. If you feel that the service cannot be inject for a reason I'm not seeing, please comment the reason utilize the SpringContext in the pattern recommended by Kuali coding standards here: https://kualico.atlassian.net/wiki/display/FINDOC/Java+Coding+Standards under the "Inject All Services" heading. If it is determined that Spring absolutely cannot generate the NoteService (unlikely), then please follow the SpringContext usage pattern in the coding standards where reference is stored as a private static variable to ensure that subclasses only use the getter.